libraries/common/subscriptions/helpers/handleLinks.js

```diff
@@ -236,18 +230,6 @@ export const handleLegacyLink = (options) => {
     });
   }
 
-  if (options.flushTab) {
-    flushTab({
```

Comment:
`flushTab` is still used in the `openLegacyLink` function. Not sure if needed.
Jsdoc can be removed if not needed.

Reply:
removed